Jellyfin.Networking/HappyEyeballs/HttpClientExtension.cs

```diff
@@ -65,7 +65,7 @@ public static async ValueTask<Stream> OnConnect(SocketsHttpConnectionContext con
             // See https://github.com/dotnet/corefx/pull/29792/files#r189415885 for more details.
             if (await Task.WhenAny(tryConnectAsyncIPv6, Task.Delay(200, cancelIPv6.Token)).ConfigureAwait(false) == tryConnectAsyncIPv6 && tryConnectAsyncIPv6.IsCompletedSuccessfully)
             {
-                cancelIPv6.Cancel();
+                await cancelIPv6.CancelAsync().ConfigureAwait(false);
```

Comment:
Why is this awaited but the next line calling `GetAwaiter().GetResult()`?

Reply:
`tryConnectAsyncIPv6` Is already completed, so `GetAwaiter().GetResult()` just gets the result